sentry_sdk/scope.py

```diff
@@ -987,8 +1099,8 @@ def continue_trace(self, environ_or_headers, op=None, name=None, source=None):
 
         return transaction
 
-    def capture_event(self, event, hint=None, client=None, scope=None, **scope_kwargs):
```

Comment:
Could we leave the `client` parameter for backwards compatibility?

Reply:
This comment also applies to the other methods in this file where `client` is removed.